pkg/repository/provider/unified_repo.go

```diff
@@ -568,6 +568,12 @@ func getStorageVariables(backupLocation *velerov1api.BackupStorageLocation, repo
 	result[udmrepo.StoreOptionOssRegion] = strings.Trim(region, "/")
 	result[udmrepo.StoreOptionFsPath] = config["fspath"]
 
+	if backupRepoConfig != nil {
+		if v, found := backupRepoConfig[udmrepo.StoreOptionCacheLimit]; found {
+			result[udmrepo.StoreOptionCacheLimit] = v
+		}
+	}
+
```

Comment:
Should we also not check for `enableCompression` config as well ?

Reply:
As our previous discussion, compression is not supported by Kopia repo in 1.15. 
From the design whether/when/how a configuration is picked is decided by the specific repo, so here it is fine for the repo not touching the configuration if it doesn't support it.

We just want to support compression as a user case in the design.
However, for implementation, we may need some more values, e.g., what is the escape size in the case that it is a waste of time/resource to do compression for small files.

Reply:
Maybe it makes sense to enable/disable compression at a backup level rather than at the backup repository level, but yeah I get your point.

Reply:
Generally speaking, compression could be at backup level, but most often, it is at repository level.
Compression is eventually done by the repository, either client part or server part. Therefore, even if we have it on backup level, we need to pass the configuration all the way down to the repository. Otherwise, server side dedup & compression will be disabled.

While for Unified Repo concepts, repository with server side dedup & compression is supported, so we will probably have it at repository level only. 

Anyway, this is out of the scope of the current PR, but a consideration point when we implement compression. Let see if we have other ideas then.